style.css

```diff
@@ -1 +1,149 @@
 /*  Add your styling here */
+*{
+    margin: 0%;
+    padding: 0%;
+}
+@import url('https://fonts.googleapis.com/css2?family=Frank+Ruhl+Libre:wght@300&family=Nanum+Gothic&family=Sarabun:wght@100&display=swap');
+
+body{
+    /* width: 390px;
+    height: 844px; */
+    
+}
+.box{
+    padding: 29px;
+    background-color: aqua;
+}
+.contener{
+display: grid;
+grid-template-columns: repeat(2,1fr);
+grid-template-rows: repeat(7,1fr) ;
+grid-gap: 3px;
+grid-template-areas: 
+"hr hr"
+"ar ar"
+"ar ar"
+"b_img b_img"
+"img1 img2"
+"img3 img4"
+"foot foot";
+}
+h1{
+    font-size: 100%;
```

Comment:
You should not use % font-sizes, you should use %for containers, and use em or px for font-size